src/schema/v1/auction_result.ts

```diff
@@ -9,12 +9,12 @@ import {
   GraphQLEnumType,
 } from "graphql"
 import { indexOf } from "lodash"
-import { connectionWithCursorInfo } from "schema/fields/pagination"
-import Image, { normalizeImageData } from "schema/image"
+import { connectionWithCursorInfo } from "schema/v1/fields/pagination"
+import Image, { normalizeImageData } from "schema/v1/image"
 import { ResolverContext } from "types/graphql"
 
 // Taken from https://github.com/RubyMoney/money/blob/master/config/currency_iso.json
-const currencyCodes = require("../lib/currency_codes.json")
+import currencyCodes from "lib/currency_codes.json"
```

Comment:
Okay, cc @alloy this change probably needs direct eyes. I'm not sure why this was a require statement, but I removed the relative import and made it fall back to the project global import. We do have some tests around this and they _are_ passing.

Reply:
Probably predates TS gaining the ability to import JSON directly.